web/spacewalk-web.spec

```diff
@@ -68,6 +68,18 @@ BuildRequires:  nodejs
 %description -n susemanager-web-libs
 This package contains Vendor bundles needed for spacewalk-web
 
+%package -n susemanager-web-libs-debug
+Summary:        Vendor bundles for spacewalk-web debug files
+License:		0BSD and BSD-3-Clause and LGPL-3.0-or-later and MIT and MPL-2.0
```

Comment:
Can you align the content of `License:` ?

Reply:
aligned: there were tabs in the summary